source/numerics.tex

```diff
@@ -13108,11 +13116,26 @@
   a.accessor())
 \end{codeblock}
 \item
+otherwise,
+\begin{codeblock}
+R(a.data_handle(), ReturnMapping(@\exposid{transpose-extents}@(a.mapping().extents()),
+  a.mapping().stride(1)), a.accessor())
+\end{codeblock}
+if \tcode{Layout} is \tcode{layout_left_padded<PaddingValue>}
+for some \tcode{size_t} value \tcode{PaddingValue};
+\item
+otherwise,
+\begin{codeblock}
+R(a.data_handle(), ReturnMapping(@\exposid{transpose-extents}@(a.mapping().extents()),
+  a.stride(0)), a.accessor())
```

Comment:
Actually, shouldn't this be:

```suggestion
  a.mapping().stride(0)), a.accessor())
```

@mhoemmen

Reply:
If we can fix this editorially now that would save going through the issue process. There is no `stride` member of `mdspan` so the approved wording is nonsensical.

Reply:
I see now, yes, makes sense. Will do!

Reply:
@jwakely @tkoeppe Hi! : - )  Good catch!  Somehow I missed that one for the `layout_right_padded` case, even though I got it right for the `layout_left_padded` case right above it.  `a.mapping().stride(0)` is absolutely what I intended to write there.

Reply:
Thanks!